lib/sparql_to_sw_solr/physical_field.rb

```diff
@@ -0,0 +1,51 @@
+module SparqlToSwSolr
+  class InstanceSolrDoc
+    module PhysicalField
+
+      private
+
+      def physical_values
+        physical_solutions.each_solution do |soln|
+          next unless soln.bindings.keys.include?(:noteType)
+          @note_type = soln.noteType.to_s
+          @physical1 = soln.extentLabel_1.to_s
+          @physical3 = soln.instanceDimensions_3.to_s
+          note_case(soln)
```

Comment:
Agreed.  The iteration needs to collect results somehow, using `Array#collect` if the `physical_solutions` supports generic Array methods, or something similar.  As noted below, I think there could be problems using instance variables in these iterations.